kotlin/samples/dungeon/app/build.gradle.kts

```diff
@@ -35,7 +35,7 @@ android {
 
   compileOptions {
     // Required for SnakeYAML.
-    coreLibraryDesugaringEnabled = true
+    isCoreLibraryDesugaringEnabled = true
```

Comment:
This was giving me a sync error without the change but perhaps it was a spurious gradle error?